kernel_corepattern_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
New files should get the current year.

```suggestion
// Copyright 2020 The Prometheus Authors
```